modules/extension/xsd/xsd-core/src/test/java/org/geotools/xsd/StreamingParserTest.java

```diff
@@ -39,4 +40,16 @@ public void testParseXXE() throws Exception {
         // StreamingParser returns null if the parsing fails
         assertNull(parser.parse());
     }
+
+    @Test
+    public void testParseWithJavaMethod() throws Exception {
+        ByteArrayInputStream in = new ByteArrayInputStream("<mails></mails>".getBytes());
+        StreamingParser parser =
+                new StreamingParser(new MLConfiguration(), in, "java.lang.Thread.sleep(30000)");
```

Comment:
I am a bit confused as to choosing a time based example? Why not ``java.lang.Math.abs(-1.0)`` - and test. to see that 1.0 is returned. This would be a test that is not sensitive to build server speed?

Reply:
This is a type of issue that is commonly referred to as `blind`. Just because something is being done doesn't mean that the result is actually sent anywhere so I needed some kind of side effect that I could check for. I originally tested this using `System.exit` but that isn't as Maven friendly.

Reply:
Okay that makes sense, thanks for explaining.